internal/controller/controllers/agent_controller.go

```diff
@@ -272,11 +404,62 @@ func (r *AgentReconciler) deregisterHostIfNeeded(ctx context.Context, log logrus
 	return buildReply(nil)
 }
 
+//
+//  In assisted installer, UserManagedNetworking implicates none platform.  This flag is part of AgentClusterInstall spec.
+//
+func (r *AgentReconciler) isNonePlatform(ctx context.Context, agent *aiv1beta1.Agent) (isNone bool, err error) {
```

Comment:
This same function is being implemented in BMAC. Can you extract it from the controllers and put it in a common place?

Reply:
2 different PRs.  They are a bit different.  I will merge them afterwards